micro-ui/web/micro-ui-internals/packages/modules/core/src/components/RoleBasedEmployeeHome.js

```
@@ -0,0 +1,172 @@
+
+  const transformURL = (url = "") => {
+    if (url == "/") {
+      return;
+    }
+    if (url?.indexOf(`/${window?.contextPath}`) === -1) {
+      let updatedUrl = null;
+      if (isMultiRootTenant) {
+        url = url.replace("/sandbox-ui/employee", `/sandbox-ui/${tenantId}/employee`);
+        updatedUrl = url;
+      } else {
+        updatedUrl = DIGIT_UI_CONTEXTS?.every((e) => url?.indexOf(`/${e}`) === -1) ? "/employee/" + url : url;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)